package.json

```diff
@@ -18,6 +19,7 @@
     "since": "./bin/plugin/cli.js since",
     "readme": "./bin/plugin/cli.js readme",
     "translations": "./bin/plugin/cli.js translations",
+    "build-plugins": "./bin/plugin/cli.js build-plugins",
```

Comment:
Once PR is ready for merge, then we will remove this.

Reply:
Why? I think it's a good  idea to keep the short-hand in here, similar to how we have for the other CLI scripts.

Reply:
Initially, I assumed it was part of the GH workflow, so I included a script for testing purposes.